### PR TITLE
Added ability to specify minOpacity

### PR DESCRIPTION
**Pull request by RobbieTheWagner**
Added an optional parameter to options to specify a minOpacity for the heat.

### CODE REVIEW

File: src/HeatLayer.js [hunk 1]
```diff
@@ -174,7 +175,7 @@ L.HeatLayer = (L.Layer ? L.Layer : L.Class).extend({
         // console.timeEnd('process');
 
         // console.time('draw ' + data.length);
-        this._heat.data(data).draw();
+        this._heat.data(data).draw(this.options.minOpacity || 0.05);
```
Comment (mourner) [inline src/HeatLayer.js]: you won't be able to specify 0 as `minOpacity`

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+node_modules/
+.idea/
```
Comment (mourner) [inline .gitignore]: No need to commit `gitignore` here. Put `node_modules` in your global `gitignore`.

File: src/HeatLayer.js (hunk 1 above)
Comment (mourner) [inline src/HeatLayer.js]: I think a better approach would be not to set default minOpacity in Leaflet.heat, but instead rely on default minOpacity of simpleheat.

Comment (RobbieTheWagner) [inline src/HeatLayer.js]: Ah, that's true. You can't currently specify anything though, so I figured it should default to, what I believe the current default is, 0.05. Should I do something like this instead?

``` js
var opacity = (this.options.minOpacity >= 0 && this.options.minOpacity <= 1) ? this.options.minOpacity : 0.05;
this._heat.data(data).draw(opacity);
```

Comment (mourner) [inline src/HeatLayer.js]: no, just `this._heat.data(data).draw(this.options.minOpacity);` should be enough I think.

**Comment by mourner** [comment]
Can you please squash the commits into one with git rebase?

**Comment by RobbieTheWagner** [comment]
@mourner I just tried multiple times to rebase or do other resets and things to squash the commits, I don't know if it worked. Nothing showed up here.

**Comment by RobbieTheWagner** [comment]
I did:
`git rebase -i HEAD~3`

Then I ended up with one commit, but when I push it to the server it just seems to disappear. What am I doing wrong?

**Comment by mourner** [comment]
Just do `git push --force`

**Comment by RobbieTheWagner** [comment]
Okay, I think I got it, sorry about the confusion.

**Comment by mourner** [comment]
Thanks!

**Comment by RobbieTheWagner** [comment]
You're welcome, thanks for merging!

**Pull request merged**